compose/ui/ui/src/desktopTest/kotlin/androidx/compose/ui/window/window/WindowTest.kt

```diff
@@ -634,4 +640,25 @@ class WindowTest {
             )
         }
     }
+
+    @Test
+    fun `pass LayoutDirection to Window`() {
+        lateinit var localLayoutDirection: LayoutDirection
+
+        var layoutDirection by mutableStateOf(LayoutDirection.Rtl)
+        rule.setContent {
+            CompositionLocalProvider(LocalLayoutDirection provides layoutDirection) {
+                Window(onCloseRequest = {}) {
```

Comment:
Let's write a test for `application -> Window -> Popup`

Reply:
I'll add one, but I don't think we can really have a bug with (for example) `Window -> Popup` if the layout direction propagates correctly into `Window` and into `Popup` (and we already have tests for these two cases).